database/ongoing/unsubscribe-users.js

```diff
@@ -0,0 +1,38 @@
+const getItems = require('../utils/get-items');
+const asyncForEach = require('../utils/async-for-each');
+const { logger, generateProgressBar } = require('../utils/logging');
+
+module.exports = {
+  async up(db, client) {
+    const emails = getItems('bouncing-emails.csv');
+    const progress = generateProgressBar(emails.length);
+    await asyncForEach(emails, async (email, index) => {
+      const log = (m, p, t = 'info') => logger(t, index, m, p);
+      log('Query out: ', email);
+      progress.tick();
+      await db
+        .collection('user')
+        .updateOne(
+          { email: email },
+          { $set: { sendQuincyEmail: false } },
+          async (err, result) => {
```

Comment:
According to the mongo [docs](https://mongodb.github.io/node-mongodb-native/3.5/api/Collection.html#updateOne) it will not return a promise if you pass it a callback.  Also, I've not found `async` callbacks being used anywhere.  Is this all `migrate-mongo` weirdness? 

I would have thought you want `await .updateOne().then.catch` type of thing.